Git/rebase.sublime-syntax

```diff
@@ -0,0 +1,55 @@
+%YAML 1.2
+---
+# Highlight interactive rebase text (git rebase -i).
+
+name: Git Rebase
+file_extensions:
+  - git-rebase-todo
+first_line_match: '^(?:pick|squash|reword|fixup|exec|edit|drop) \h{7,} '
+scope: source.gitrebase
+
+variables:
+  hash: \h{7,}
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - include: commit
+
+  comments:
+    - match: '#'
+      scope: punctuation.definition.comment.gitrebase
+      push:
+        - meta_scope: comment.line.gitrebase
+        - match: \n
+          pop: true
+        - include: rebase-msg
+
+  # e.g. pick d284bb2 Initial commit
+  commit:
+    - match: ^\s*(squash|reword|pick|fixup|exec|edit|drop|x|s|r|p|f|e|d)\s
+      captures:
+        1: storage.type.edit.gitrebase
+      push:
+        - meta_scope: meta.commit.gitrebase
+        - match: \n
+          pop: true
+        - match: \b({{hash}})\s(.*) # e.g. d284bb2
```

Comment:
I'm always weary of these "match *anything*" regexes. Maybe just match `\b{{hash}}\b` and then push onto a new context?